embeddable-say-editor/app/components/simple-editor.hbs

```diff
@@ -137,6 +137,20 @@
                 @config={{this.config.citation}}
               />
             {{/if}}
+            {{#if this.uiConfig.showRdfaEditor}}
```

Comment:
Since these UI elements are only related to, and require, the 'rdfa-editor' plugin, I think we should just use `(array-includes this.activePlugins 'rdfa-editor')`, since that's what we're using for the editor options flags anyway. I think the idea of the `uiConfig` object was that it is for any configuration that could be switched on or off by any plugin, for example, whether to show the sidebar.